src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -50,6 +50,7 @@ public sealed class VsSolutionRestoreService : IVsSolutionRestoreService, IVsSol
         private const string RestoreSources = nameof(RestoreSources);
         private const string RestoreFallbackFolders = nameof(RestoreFallbackFolders);
         private const string AssetTargetFallback = nameof(AssetTargetFallback);
+        private const string RestoreAdditionalProjectFallbackFoldersExcludes = nameof(RestoreAdditionalProjectFallbackFoldersExcludes);
         private const string RestoreAdditionalProjectFallbackFolders = nameof(RestoreAdditionalProjectFallbackFolders);
         private const string RestoreAdditionalProjectSources = nameof(RestoreAdditionalProjectSources);
```

Comment:
I considered this, but I don't there would be a benefit. There isn't a scenario where both VS and the command line restore are being used at the same time, you would load one or the other, hence no memory saved.